### PR TITLE
New: Default RSS Sync Interval to 30 minutes

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
- Default RSS to ~~15min~~ 30min from 60min for new installs
- Improve user experience
- aligns with upstream

#### Todos
- [x] Tests
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
-  [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes missed RSS grabs for new users

### CODE REVIEW

**Comment by bakerboy448** [comment]
I'll look forward to the PR upstream increasing the time to more than 15 minutes then - https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/Configuration/ConfigService.cs#L108

**Comment by bakerboy448** [comment]
suggestions for timing welcome....and should this go upstream? cc @markus101

**Comment by bakerboy448** [comment]
> Considering the frequency of new releases of movies vs episodes, I don't think they're the same.

The majority of the RSS feeds from sites are not filtered by category so the same quantity of releases needs to be processed and this point is moot.

Hell even 15 minutes is often frequently having gaps
image

**Comment by markus101** [comment]
15 minutes is already the default is Sonarr, which for most sites is the minimum they allow.

Not being able to filter by category sounds like something indexers will need to address, I don't have these issues with the indexers I'm using.

**Comment by Qstick** [comment]
Filtering by category for usenet indexers is in spec is it not? Maybe if you are using pirate bay through prowlarr or something similar I can see that popping up.

File: src/NzbDrone.Core/Configuration/ConfigService.cs [hunk 1]
```diff
@@ -105,7 +105,7 @@ public int RecycleBinCleanupDays
 
         public int RssSyncInterval
         {
-            get { return GetValueInt("RssSyncInterval", 60); }
+            get { return GetValueInt("RssSyncInterval", 15); }
```
Comment (Qstick) [inline src/NzbDrone.Core/Configuration/ConfigService.cs:108]:
```suggestion
            get { return GetValueInt("RssSyncInterval", 30); }
```

**Review by Qstick — CHANGES_REQUESTED** [review]
Fix the tests

**Comment by Qstick** [comment]
Just a squash

**Review by Qstick — APPROVED** [review]

**Pull request merged**